system-variables.md

```diff
@@ -795,6 +795,18 @@ Constraint checking is always performed in place for pessimistic transactions (d
 >
 > Currently, this feature is incompatible with TiDB Binlog in some scenarios and might cause semantic changes on a transaction. For more usage precautions of this feature, refer to [Incompatibility issues about transaction semantic](https://github.com/pingcap/tidb/issues/21069) and [Incompatibility issues about TiDB Binlog](https://github.com/pingcap/tidb/issues/20996).
 
+### tidb_enable_analyze_snapshot <span class="version-mark">New in v6.2.0</span>
```

Comment:
The description of `tidb_enable_paging` is inconsistent with https://github.com/pingcap/docs-cn/pull/10679

Reply:
https://github.com/pingcap/docs-cn/pull/10679 is reverted and does not need translation.